Jenkinsfile

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env groovy
 
 common {
+    dockerPush = true
+    dockerRepos = ['confluentinc/cp-base']
+    nodeLabel = 'docker-oraclejdk7'
```

Comment:
Looks like dockerfile is hardcode to jdk8. Any reason that it shouldn't / can't use defaultNodeLabel? That will help us manage the labels easier. 

ksql 4.1.x is using dockerfile (jdk8) while other 4.1.x are using jdk7, and so that's actually one place where it makes it kinda hard to migrate/transition from old images to new images as there is no clean split between jdk7 and jdk8 based on branch. And you mentioned that we won't be using Maven to build Docker images in the future, which makes the problem more complicated / not sure how things should work.

Reply:
I'm just copying over the attributes from common, trying to change as little as possible.

I prefer to twiddle knobs in a separate iteration as needed.

Reply:
I prefer to change little as well, but something that isn't necessary should be fixed, such as the nodeLabel for this branch (4.1.x). defaultNodeLabel defaults to JDK7 for 4.1.x or lower, and JDK8 for higher, so it will use the same label here / there is no need to explicit set.

Also, by switching dockerfile to use defaultNodeLabel, it will also solve our other problem where we mix of jdk7 vs 8 in 4.1.x -- assuming jdk7 will work for ksql 4.1.x (if not, then it makes it a bit harder for us to build things consistently).  Even if not, we can explicitly set the label for ksql where there is a need rather than other repos (this one) that does not need to be hard coded. This is mostly a problem with how dockerfile is setup / not using defaultNodeLabel, so ok to be done in a separate change / later.